source/plugin/internals/mcp-setup.rst

```diff
@@ -21,9 +21,9 @@ You can choose between two different types of workspaces:
     problems on one of the platforms because of changes in the Minecraft code by Forge. Make sure to always test your
     plugin on both platforms when using MCP.
 
-Choosing a MCP mappings version
-```````````````````````````````
-To setup a MCP workspace you need to specify the MCP mappings version that will be used to de-obfuscate the Minecraft
+Choosing an MCP mappings version
```

Comment:
"a MCP" was correct afaik, as it's an acronym (Mod Coder Pack). I guess "an" will suffice if you say it "Emseepee".

Reply:
At first i wondered my grammar checker asked me to change it, but then google translate said the same.

My eyes say it is wrong with an. My mouth says its right. I will use whatever you prefer.

Reply:
http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html
`emm-see-pee`, so "an" is correct

Reply:
Fair enough; I'm used to just mentally substituting "Mod Code Pack" instead of the acronym.